spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -258,6 +258,9 @@ A portable extension or other object may obtain a reference to the current conta
 Other methods on `CDI` may be called after the application initialization is completed until the application shutdown starts.
 If methods on `CDI` are called at any other time, non-portable behavior results.
 
+CDI implements `javax.enterprise.inject.Instance` and therefore might be used to perform programmatic lookup as defined in <<dynamic_lookup>>.
+If no qualifier is specified in `Instance.select()` method the `@Default` qualifier is assumed.
```

Comment:
Wouldn't be more clear to explicitly mention `CDI.current().select()` instead of `Instance.select()`? It's just minor....